app/components/clientsideonly/fuselage.js

```diff
@@ -0,0 +1,70 @@
+import dynamic from "next/dynamic";
```

Comment:
Let's move this to `inappchat/lib/fuselage.js`

Reply:
done